examples/apps/release-header/src/main.rs

```diff
@@ -108,93 +62,97 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     result
 }
 
-fn run(mut terminal: DefaultTerminal) -> Result<(), Box<dyn std::error::Error>> {
+fn run(mut terminal: DefaultTerminal) -> color_eyre::Result<()> {
     loop {
-        terminal.draw(render_header)?;
-        if ratatui::crossterm::event::read()?.is_key_press() {
+        terminal.draw(render)?;
+        if event::read()?.is_key_press() {
             break Ok(());
         }
     }
 }
 
-fn render_header(frame: &mut Frame) {
+fn render(frame: &mut Frame) {
     let area = frame.area();
-    frame
-        .buffer_mut()
-        .set_style(area, Style::new().fg(FG_COLOR).bg(BG_COLOR));
-
-    let horizontal_layout = Layout::horizontal([Constraint::Length(31), Constraint::Length(23)])
-        .spacing(Spacing::Overlap(1));
-    let [left_area, right_area] = area
-        .centered(
-            Constraint::Length(53),
-            Constraint::Length(MAIN_DISHES.len() as u16 + BACKENDS.len() as u16 + 3),
-        )
-        .layout(&horizontal_layout);
-
-    render_logo(frame, left_area);
-    render_menu(frame, right_area);
+    frame.buffer_mut().set_style(area, (FG_COLOR, BG_COLOR));
+
+    let logo_width = 29;
+    let menu_width = 23;
+    let padding = 2; // Padding between logo and menu
+    let menu_borders = 3;
+    let height = MAIN_DISHES.len() as u16 + BACKENDS.len() as u16 + menu_borders;
+    let width = logo_width + menu_width + padding;
+    let center_area = area.centered(Constraint::Length(width), Constraint::Length(height));
+    let layout = Layout::horizontal(Constraint::from_lengths([logo_width, padding, menu_width]));
+    let [logo_area, _, menu_area] = center_area.layout(&layout);
+
+    render_logo(frame, logo_area);
+    render_menu(frame, menu_area);
 }
 
 fn render_logo(frame: &mut Frame, area: Rect) {
-    let logo_block = Block::new().padding(Padding::new(1, 1, 1, 0));
-    let logo_inner_area = logo_block.inner(area);
-    let vertical_layout = Layout::vertical([
-        Constraint::Length(2),
-        Constraint::Fill(1),
-        Constraint::Length(1),
-    ]);
-    let [logo_area, _, version_area] = logo_inner_area.layout(&vertical_layout);
-
-    render_logo_rainbow_shadow(frame, logo_area);
-
-    let logo_offset = Offset::new(0, 6);
-    frame.render_widget(Block::new().style(FG_COLOR), logo_area.offset(logo_offset));
-    frame.render_widget(RatatuiLogo::small(), logo_area.offset(logo_offset));
-
-    frame.render_widget(
-        Line::from(format!("v{SEMVER} \"{RELEASE_NAME}\"")).dim(),
-        version_area,
-    );
+    let area = area.inner(Margin::new(1, 0));
+    let layout = Layout::vertical(Constraint::from_lengths([6, 2, 1])).flex(Flex::End);
```

Comment:
Incidentally, this also looks nice with only 5 shadows and moved up one line now that the extra text is added for the code name. It could be worth considering this option...

<img width="686" height="331" alt="image" src="https://github.com/user-attachments/assets/bb10d90e-abe6-4f3c-8d89-6be52188d553" />